integrations/to.etc.domui.hibutil/src/main/java/to/etc/domui/hibernate/config/HibernateConfigurator.java

```diff
@@ -446,6 +447,8 @@ public synchronized static void initialize(final DataSource ds) throws Exception
 		}
 
 		m_contextSource = new HibernateLongSessionContextFactory(m_listeners, hsm, m_handlers, HibernateConfigurator::onContextCreated);
+		ExceptionDialog.register(HibernateMessageDecoder::translateHibernateException);
```

Comment:
No idea how this ended up here, I guess that is something I picked up with some merges? It does look like some valid change, so I'll assume it is ok?